generator-templates/component/test.hbs

```diff
@@ -1,3 +1,4 @@
+import React from 'react'
```

Comment:
Tests that use this template didn't work without a React import, so I added it to the hbs